src/index.ts

```diff
@@ -189,7 +189,7 @@ function runServer(schemaIDL: Source, extensionIDL: Source, optionsCB) {
     });
   });
 
-  app.use('/user-idl', bodyParser.text());
+  app.use('/user-idl', bodyParser.text({limit: '5mb', extended: true}));
```

Comment:
@arquio Thanks for PR 👍 
I understand the need for `limit` but can you please explain why you need `extended`?

Reply:
nit: can you set a limit to some power of 2 like `8` or `16`
since if we add magic numbers adding power of 2 ones looks a little bit more professional 😄

Reply:
8mb and deleted extended, its no necesary.

thanks!!